ui/.ember-cli

```diff
@@ -5,6 +5,5 @@
 
     Setting `disableAnalytics` to true will prevent any data from being sent.
   */
-  "disableAnalytics": false,
-  "proxy": "http://127.0.0.1:4646"
+  "disableAnalytics": false
```

Comment:
eep sorry, I meant to explain this. Just as I was starting to prototype this, [I found that](https://hashicorp.slack.com/archives/C805RSR2N/p1573578005406200?thread_ts=1573251854.383800&cid=C805RSR2N) I was completely unable to connect to a websocket through that proxy because the `Origin` didn’t match and the request was rejected by the API. The Javascript socket interface doesn’t give you any control over that for security reasons, so I switched to the technique described [here](https://blog.201-created.com/secrets-of-the-ember-cli-server-bde80bb546dd) because it gives you fine-grained control. The crux for this feature is [this section](https://github.com/hashicorp/nomad/pull/6697/files#diff-867738bf87da842827bf279061902df2R30) where the origin gets rewritten in the proxy outside the browser. It’s a shame to lose this simplicity but it seems unavoidable :////////

Reply:
Wow. TIL. Definitely a shame when things don't just work, but given the amount of oddities in our use of HTTP, I'm surprised the default proxy even got us this far.